rebar.config

```diff
@@ -19,3 +19,8 @@
     {gleam_otp, "~> 0.1"},
     elli
 ]}.
+
+{relx, [{release, {echo, "0.1.0"},
+         [echo, sasl, gleam_otp, gleam_http, gleam_stdlib, gleam_elli]},
```

Comment:
Is it not enough to only specify `echo` and have the others be pulled in as its deps?

Reply:
If I only have `echo` this I get this as output in the build:
```
===> There are missing function calls in the release.                                                                                      
===> Make sure all applications needed at runtime are included in the release.                                                             
===> echo@application:init/1 calls undefined function gleam@otp@supervisor:add/2                                                           
===> echo@application:init/1 calls undefined function gleam@otp@supervisor:worker/1                                                        
===> echo@application:start/2 calls undefined function gleam@otp@process:pid/1                                                             
===> echo@application:start/2 calls undefined function gleam@otp@supervisor:start/1                                                        
===> echo@application:start/2 calls undefined function gleam@result:map/2                                                                  
===> echo@application:stop/1 calls undefined function gleam@otp@supervisor:application_stopped/0                                           
===> echo@web:echo/1 calls undefined function gleam@http:get_req_header/2                                                                  
===> echo@web:echo/1 calls undefined function gleam@http:prepend_resp_header/3                                                             
===> echo@web:echo/1 calls undefined function gleam@http:response/1                                                                        
===> echo@web:echo/1 calls undefined function gleam@http:set_resp_body/2                                                                   
===> echo@web:echo/1 calls undefined function gleam@result:unwrap/2                                                                        
===> echo@web:hello/1 calls undefined function gleam@bit_string:from_string/1                                                              
===> echo@web:hello/1 calls undefined function gleam@http:prepend_resp_header/3                                                            
===> echo@web:hello/1 calls undefined function gleam@http:response/1                                                                       
===> echo@web:hello/1 calls undefined function gleam@http:set_resp_body/2                                                                  
===> echo@web:hello/1 calls undefined function gleam@string:concat/1                                                                       
===> echo@web:hello/1 calls undefined function gleam@string:lowercase/1                                                                    
===> echo@web:not_found/0 calls undefined function gleam@bit_string:from_string/1                                                          
===> echo@web:not_found/0 calls undefined function gleam@http:prepend_resp_header/3                                                        
===> echo@web:not_found/0 calls undefined function gleam@http:response/1                                                                   
===> echo@web:not_found/0 calls undefined function gleam@http:set_resp_body/2                                                              
===> echo@web:service/1 calls undefined function gleam@http:path_segments/1                                                                
===> echo@web:start/0 calls undefined function gleam@bit_builder:from_bit_string/1                                                         
===> echo@web:start/0 calls undefined function gleam@http@elli:start/2                                                                     
===> echo@web:start/0 calls undefined function gleam@http@middleware:map_resp_body/2                                                       
===> echo@web:start/0 calls undefined function gleam@http@middleware:prepend_resp_header/3                                                 
===> echo@web@logger:format_log_line/2 calls undefined function gleam@http:method_to_string/1
===> echo@web@logger:format_log_line/2 calls undefined function gleam@int:to_string/1
===> echo@web@logger:format_log_line/2 calls undefined function gleam@string:uppercase/1
===> echo@web@logger:format_log_line/2 calls undefined function gleam@string_builder:append/2
===> echo@web@logger:format_log_line/2 calls undefined function gleam@string_builder:from_string/1
===> echo@web@logger:format_log_line/2 calls undefined function gleam@string_builder:to_string/1
===> echo@web@logger:middleware/1 calls undefined function gleam@io:println/1
===> Warnings generating release:
*WARNING* Missing application sasl. Can not upgrade with this release 
===> Release successfully assembled: _build/default/rel/echo
```
Perhaps it is still ok but I assumed not?

Reply:
And running it fails like this:

```
=CRASH REPORT==== 27-Nov-2020::19:24:38.931095 ===                                                                                         
  crasher:                                                                                                                                 
    initial call: application_master:init/4                                                                                                
    pid: <0.239.0>                                                                                                                         
    registered_name: []                                                                                                                    
    exception exit: {bad_return,                                                                                                           
                     {{echo@application,start,[normal,[]]},                                                                                
                      {'EXIT',                                                                                                             
                       {undef,                                                                                                             
                        [{gleam@otp@supervisor,start,                                                                                      
                          [#Fun<echo@application.1.29245338>],                                                                             
                          []},                                                                                                             
                         {echo@application,start,2,
                          [{file,"/src/gen/src/echo@application.erl"},
                           {line,14}]},
                         {application_master,start_it_old,4,
                          [{file,"application_master.erl"},{line,277}]}]}}}}
      in function  application_master:init/4 (application_master.erl, line 138)
    ancestors: [<0.238.0>]
    message_queue_len: 1
    messages: [{'EXIT',<0.240.0>,normal}]
    links: [<0.238.0>,<0.200.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 610
    stack_size: 28
    reductions: 213
  neighbours:
=INFO REPORT==== 27-Nov-2020::19:24:38.931813 ===
    application: echo
    exited: {bad_return,
                {{echo@application,start,[normal,[]]},
                 {'EXIT',
                     {undef,
                         [{gleam@otp@supervisor,start,
                              [#Fun<echo@application.1.29245338>],
                              []},
                          {echo@application,start,2,
                              [{file,"/src/gen/src/echo@application.erl"},
                               {line,14}]},
                          {application_master,start_it_old,4,
                              [{file,"application_master.erl"},
                               {line,277}]}]}}}}
    type: permanent

{"Kernel pid terminated",application_controller,"{application_start_failure,echo,{bad_return,{{echo@application,start,[normal,[]]},{'EXIT',{undef,[{gleam@otp@supervisor,start,[#Fun<echo@application.1.29245338>],[]},{echo@application,start,2,[{file,\"/src/gen/src/echo@application.erl\"},{line,14}]},{application_master,start_it_old,4,[{file,\"application_master.erl\"},{line,277}]}]}}}}}"}
Kernel pid terminated (application_controller) ({application_start_failure,echo,{bad_return,{{echo@application,start,[normal,[]]},{'EXIT',{undef,[{gleam@otp@supervisor,start,[#Fun<echo@application.1.2

Crash dump is being written to: erl_crash.dump...done
```
None of this means much to me though :)

Reply:
I guess it is required. Seems frustrating.